geometry_msgs/Pose pose
float32 magnetic_force
